app/internal/translation.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from iso639 import languages
+from sqlalchemy.exc import SQLAlchemyError
+from textblob import TextBlob, download_corpora
+from textblob.exceptions import NotTranslated
+
+from app.database.database import SessionLocal
+from app.database.models import User
+
+download_corpora.download_all()
+
+
+def translate_text(text: str,
+                   target_lang: str,
+                   original_lang: Optional[str] = None
+                   ) -> str:
+    """
+    Translate text to the target language
+    optionally given the original language
+    """
+    if len(text) <= 0:
```

Comment:
Maybe check for empty string ("") or None?

Reply:
Yes, this check is not good enough, but your version also does not check if it is an empty string. I changed the check so that it doesn't skip spaces either. Thank you for paying attention to this.

Reply:
You can use `if text.strip():`, which checks for empty strings or strings with only spaces :)

Reply:
yes that's exactly what i did  :)